sdk/loadtestservice/Azure.Developer.LoadTesting/tests/Samples/Sample1_CreateOrUpdateTest.cs

```diff
@@ -19,22 +14,17 @@ public partial class LoadTestingSamples: SamplesBase<LoadTestingClientTestEnviro
         [SyncOnly]
         public void CreateOrUpdateTest()
         {
-            #region Snippet:Azure_Developer_LoadTesting_CreatingClient
+            #region Snippet:Azure_Developer_LoadTesting_CreateAdminClient
 
             string endpoint = TestEnvironment.Endpoint;
+            Uri enpointUrl = new Uri("https://"+endpoint);
```

Comment:
TestEnvironment shouldn't be in the sample code. Use `#if SNIPPET` blocks to include code in the markdown but have it not compile in the cs file.

Reply:
please fix in all other samples.